### DISCUSSION TITLE
Meta Title auto option should display the entire "composed" title

### DISCUSSION THREAD

**Discussion by el-schneider** [aerni/statamic-advanced-seo#91] [Ideas]
If I have a `Site Title` set in `/cp/advanced-seo/site/general` and my meta title is on `auto` it will only display the current entry title but not the composed version, that will actually be used in the `<title>` 

I think it would be a nice addition and better UX if this could be added

<img width="910" alt="grafik" src="https://github.com/aerni/statamic-advanced-seo/assets/26460248/5fe41600-dfe3-4d9a-943b-49046a8c2640">

**Comment by aerni** [comment]
I agree and have been thinking about improving the UI before. Adding it inside the text field doesn't feel like a good solution, though. To make the user experience consistent, you'd also need to see the site title when `Custom` is selected. The best solution is probably to have a computed value somewhere below the text field.

**Comment by el-schneider** [comment]
Maybe pulling the position setting up top and doing it yoast-style like this could work:

<img width="737" alt="grafik" src="https://github.com/aerni/statamic-advanced-seo/assets/26460248/d6f396ef-6221-473f-be75-cb305785f8cb">

**Comment by aerni** [comment]
Yes, I have been thinking about something along those lines. But holding off right now, as Google is rapidly changing with the introduction of Bard. Search results might not look the same a year from now. So another UI solution that isn't imitating a search engine might be better.

**Comment by el-schneider** [comment]
I agree that search will likely change quite a bit, but for the time beeing something that resembles a search engine result still might prove useful, because it also communicates to the non-technical people, what a meta description is useful for. If the ten blue links go away, maybe seo plugins become useless too, so having a interface for the next year or so is not too bad 🤗 

I woudn't mind if it became less "googley" though. I can come up with another mockup if you want.

**Comment by aerni** [comment]
Yes, some sort of interface makes sense for sure. But it's a lot of work and effort. So not making this decision lightly.
